SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialWorldSettings.cpp

```diff
@@ -42,6 +36,11 @@ TSubclassOf<USpatialMultiWorkerSettings> ASpatialWorldSettings::GetMultiWorkerSe
 		// behaviour).
 		return USpatialMultiWorkerSettings::StaticClass();
 	}
+	else if (!IsMultiWorkerEnabled())
```

Comment:
I don't think we need `ASpatialWorldSettings::IsMultiWorkerEnabled` anymore. It only returns false if the editor config has been set, so we should remove this function, and put it into `USpatialStatics::IsSpatialMultiWorkerEnabled`

Reply:
Of course, that is a neater solution.